packages/expo-notifications/ios/EXNotifications/Permissions/EXUserFacingNotificationsPermissionsRequester.m

```diff
@@ -104,21 +95,14 @@ - (void)requestPermissions:(NSDictionary *)permissions withResolver:(EXPromiseRe
   if ([permissions[@"allowDisplayInCarPlay"] boolValue]) {
     options |= UNAuthorizationOptionCarPlay;
   }
-  if (@available(iOS 12.0, *)) {
-    if ([permissions[@"allowCriticalAlerts"] boolValue]) {
-        options |= UNAuthorizationOptionCriticalAlert;
-    }
-    if ([permissions[@"provideAppNotificationSettings"] boolValue]) {
-        options |= UNAuthorizationOptionProvidesAppNotificationSettings;
-    }
-    if ([permissions[@"allowProvisional"] boolValue]) {
-        options |= UNAuthorizationOptionProvisional;
-    }
+  if ([permissions[@"allowCriticalAlerts"] boolValue]) {
+      options |= UNAuthorizationOptionCriticalAlert;
   }
-  if (@available(iOS 13.0, *)) {
-    if ([permissions[@"allowAnnouncements"] boolValue]) {
-      options |= UNAuthorizationOptionAnnouncement;
```

Comment:
It was introduced in iOS 13, but deprecated and ignored as of iOS 15, so I just removed this property. It was already marked as deprecated in our docs, so that should be fine.